src/pages/Online/index.js

```diff
@@ -0,0 +1,48 @@
+import { useNavigate } from "react-router-dom"; 
+
+const OnlineShop = () => {
+
+    const navigate = useNavigate();
+
+    const showResult = () => {
+        navigate('/searchResults', {replace: true});
+    }
+
+    return (
+        <div>
+            <form>
+                <div class="mb-6  mx-auto">
+                    <label for="text" class="block mb-2 text-6xl font-medium text-gray-900 dark:text-white">api_key</label>
+                    <input type="text" id="email" class="bg-gray-50 border border-gray-300 text-gray-900 text-4xl rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500" required></input>
+                </div>
+                <div class="mb-6 mx-auto">
+                    <label for="text" class="block mb-2 text-6xl font-medium text-gray-900 dark:text-white">secret</label>
+                    <input type="text" id="password" class="bg-gray-50 border border-gray-300 text-gray-900 text-4xl rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500" required></input>
+                </div>
+                <div class="mb-6 mx-auto">
+                    <label for="text" class="block mb-2 text-6xl font-medium text-gray-900 dark:text-white">name</label>
+                    <input type="text" id="password" class="bg-gray-50 border border-gray-300 text-gray-900 text-4xl rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500" required></input>
+                </div>
+                <div class="mb-6 mx-auto">
+                <label for="countries" class="block mb-2 text-6xl font-medium text-gray-900 dark:text-white">Select an brand</label>
+                    <select id="countries" class="bg-gray-50 border border-gray-300 text-gray-900 text-4xl rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500">
```

Comment:
these options should come from backend, plz invoke api call here to get the list of brand